<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmpruntManager</class>
 <widget class="QWidget" name="EmpruntManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(221, 236, 255);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>100</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>ID Membre</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QComboBox" name="comboBoxIDMembre"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>ID Livre</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0" colspan="2">
               <widget class="QComboBox" name="comboBoxIdLivre"/>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Date d'emprunt</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0" colspan="2">
               <widget class="QDateEdit" name="lineDateEmprunt">
                <property name="dateTime">
                 <datetime>
                  <hour>0</hour>
                  <minute>0</minute>
                  <second>0</second>
                  <year>2024</year>
                  <month>10</month>
                  <day>11</day>
                 </datetime>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Date de retour prevue</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0" colspan="2">
               <widget class="QDateEdit" name="lineDateRetourPrevue">
                <property name="dateTime">
                 <datetime>
                  <hour>0</hour>
                  <minute>0</minute>
                  <second>0</second>
                  <year>2024</year>
                  <month>10</month>
                  <day>11</day>
                 </datetime>
                </property>
               </widget>
              </item>
              <item row="9" column="0" colspan="2">
               <widget class="QWidget" name="widget" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QPushButton" name="btnEnregistrer">
                   <property name="text">
                    <string>Enregistrer Emprunt</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="10" column="0" colspan="2">
               <widget class="QWidget" name="widget_2" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item alignment="Qt::AlignHCenter">
                  <widget class="QLabel" name="labelInfo">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="11" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Date de retour r√©elle</string>
                </property>
               </widget>
              </item>
              <item row="12" column="0" colspan="2">
               <widget class="QDateEdit" name="lineDateRetourReelle">
                <property name="dateTime">
                 <datetime>
                  <hour>0</hour>
                  <minute>0</minute>
                  <second>0</second>
                  <year>2024</year>
                  <month>10</month>
                  <day>11</day>
                 </datetime>
                </property>
               </widget>
              </item>
              <item row="13" column="0" colspan="2">
               <widget class="QPushButton" name="btnEnregistrerRetour">
                <property name="text">
                 <string>Enregistrer retour</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QTableView" name="tableEmprunt">
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustIgnored</enum>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>160</number>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="btnCharger">
          <property name="text">
           <string>Charger la liste</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
